crates/ty_python_semantic/resources/mdtest/scopes/nonlocal.md

```diff
@@ -43,3 +43,272 @@ def f():
         def h():
             reveal_type(x)  # revealed: Unknown | Literal[1]
 ```
+
+## The `nonlocal` keyword
+
+Without the `nonlocal` keyword, `x += 1` is not allowed in an inner scope, even if we break it up
```

Comment:
"`x += 1` is not allowed in an inner scope" feels a bit confusing to me here -- this would obviously be allowed, because here `x` is unambiguously a local variable within the scope of `g`:

```py
def f():
    x = 1
		def g():
		    x = 1
		    x += 1
```

Reply:
d473f0e1dcb27d3b4cee0f648b5cde241c762819